operator/src/main/resources/scripts/introspectDomain.py

```diff
@@ -99,6 +99,8 @@
 from utils import *
 from weblogic.management.configuration import LegalHelper
 
+ISTIO_NAP_NAMES = ['tcp-cbt', 'tcp-ldap', 'tcp-iiop', 'tcp-snmp', 'http-default', 'tcp-default', 'https-secure', 'tls-ldaps', 'tls-default', 'tls-cbts', 'tls-iiops', 'https-admin']
+WLS_LOCALHOST_IDENTIFIER = '-lh'
```

Comment:
i'm not sure if the WLS_LOCALHOST_IDENTIFIER '-lh' value is unique enough for the `nameContainsLocalHostIdentifier` code below, which uses this value to distinguish localhost channels from other types of channels). 

just brain-storming, but maybe instead follow the pattern of other generated channels, where IIRC, we fail introspection if we detect a customer has used a reserved name/prefix?  we could start all localhost channels with 'lhs-' _and_ have a check if customer prefixed their own channel starting with 'lhs-'...

Reply:
or maybe it'd be 'good enough' to advance the `nameContainsLocalHostIdentifier` to look for '-lhNN' instead of just '-lh'....

Reply:
change LGTM